Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -282,4 +289,30 @@ const std::string GameListItem::GetWiiFSPath() const
 
 	return ret;
 }
-
+bool GameListItem::isElfDol(const GameListItem& game)
```

Comment:
Are you sure? A string can't call GetName() on it like in your example above. Won't that fail?